packages/destination-actions/src/destinations/fullstory/identifyUser/index.ts

```diff
@@ -0,0 +1,121 @@
+import type { ActionDefinition } from '@segment/actions-core'
+import camelCase from 'lodash/camelCase'
+import type { Settings } from '../generated-types'
+import { setUserPropertiesRequestParams } from '../request-params'
+
+const action: ActionDefinition<Settings> = {
```

Comment:
This definition was largely copied from the FullStory browser destination, tweaking as seemed sensible (e.g. setting platform to cloud). The `perform` action has been updated to use the FullStory HTTP API instead of the FS client API.